packages/vx-demo/src/pages/docs/Grid.tsx

```diff
@@ -7,10 +7,11 @@ import DocPage from '../../components/DocPage';
 import AxisTile from '../../components/Gallery/AxisTile';
 import BarStackTile from '../../components/Gallery/BarStackTile';
 import ThresholdTile from '../../components/Gallery/ThresholdTile';
+import GridTile from '../../components/Gallery/GridTile';
 
 const components = [GridRows, GridColumns, Grid];
 
-const examples = [AxisTile, BarStackTile, ThresholdTile];
+const examples = [AxisTile, BarStackTile, ThresholdTile, GridTile];
```

Comment:
Instead of adding a whole new tile for this, could we instead demo the functionality in an existing example like `/axis`?. posting a screenshot of the change would be ideal, too! ✨